src/policy/copyspace.rs

```diff
@@ -125,6 +125,15 @@ impl<VM: VMBinding> CopySpace<VM> {
 
     pub fn prepare(&self, from_space: bool) {
         self.from_space.store(from_space, Ordering::SeqCst);
+        if let MetadataSpec::OnSide(side_forwarding_status_table) =
```

Comment:
This is performance-critical. We may switch to some sort of "circular" forwarding status in the future to avoid this bulk zeroing.